### ISSUE TITLE
[BUG] Portfolio Page - html / css broken - page displays improperly

### ISSUE THREAD

**Issue by compumatter** [Leantime/leantime#1262]
*If you have problems with the installation please use our [community forum](https://community.leantime.io)*

Click the "My Portfolio" (briefcase) icon in the top bar and the page displays improperly.

I have also noticed a fair number of errors in the browser developers console.

This issue happens in multiple browsers. 

![image](https://user-images.githubusercontent.com/121611/211128824-79950975-464b-419d-b38a-674335a35482.png)
**Leantime Version**
Which Leantime version are you using
2.3.3

**Server**
NGINX

**PHP / MySQL Version**
PHP 8.1 / Mysql 6.5

I have created a video to help make the problem more obvious:
* https://www.youtube.com/watch?v=pVIBG-YbXaE

**Comment by compumatter** [comment]
Something else I just noticed.  A new 'fresh installation' does not result in a broken page.  Only my upgrade.  I have upgraded many times without incident.  I have carefully documented that process as the attached screenshot shows 

However in looking at the leantime **/logs/error.log** I see the following errors returned:
![image](https://user-images.githubusercontent.com/121611/211233473-6d872641-d146-46dd-a628-9994042c1bbe.png)

I hope that helps.

Upgrade Procedure Documented:
![image](https://user-images.githubusercontent.com/121611/211233266-5563b680-8028-49df-bf15-c098477a066e.png)
